Story-Chief/frontend-test-kharlamenkomatvii
# fix: sotring for team event calendar

## Description

I changed sorting of checking type to see all events. Instead of searching for one event with all types, we are looking for events with types that we selected 

## Testing Instructions

1. Select different event types in team calendar.
2. Check that we can see all events with selected type.

## Screenshots


<img width="1207" alt="Screenshot 2023-05-26 at 11 25 38" src="https://github.com/Story-Chief/frontend-test-kharlamenkomatvii/assets/104762677/f5c0d903-6221-48e9-bc69-e74c8291dcee">
<img width="1196" alt="Screenshot 2023-05-26 at 11 25 24" src="https://github.com/Story-Chief/frontend-test-kharlamenkomatvii/assets/104762677/0d8120e9-1673-489e-a71c-6772da9e236b">
